main.cpp

```diff
@@ -345,18 +345,18 @@ static const char *execution_model_to_str(spv::ExecutionModel model)
 		return "fragment";
 	case ExecutionModelGLCompute:
 		return "compute";
-	case ExecutionModelRayGenerationNV:
-		return "raygenNV";
-	case ExecutionModelIntersectionNV:
-		return "intersectionNV";
-	case ExecutionModelCallableNV:
-		return "callableNV";
-	case ExecutionModelAnyHitNV:
-		return "anyhitNV";
-	case ExecutionModelClosestHitNV:
-		return "closesthitNV";
-	case ExecutionModelMissNV:
-		return "missNV";
+	case ExecutionModelRayGenerationKHR:
```

Comment:
KHR is still provisional, and we should not ship anything based on this extension until it is fully ratified. For now, please use NV. The enums are basically the same anyways.

Reply:
Done